### PR TITLE
Update debugging-runtime.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
* MSFT style guide, e.g. avoid future tense.
* Less is more
* Recommended approach first.

### CODE REVIEW

File: docs/workflow/debugging/coreclr/debugging-runtime.md [hunk 1]
```diff
@@ -223,15 +223,19 @@ Native C++ code is not everything in our runtime. Nowadays, there are lots of st
 
 #### Resolving Signature Validation Errors in Visual Studio
 
-Starting with Visual Studio 2022 version 17.5, Visual Studio will validate that the debugging libraries that shipped with the .NET Runtime are correctly signed before loading them. If they are unsigned, Visual Studio will show an error like:
+Visual Studio 2022 version 17.5 and later validates that the debugging libraries that shipped with the .NET Runtime are signed before loading them. If they are unsigned, Visual Studio shows an error similar to the following:
```
Comment (Rick-Anderson) [inline docs/workflow/debugging/coreclr/debugging-runtime.md:226]:
```suggestion
Visual Studio 2022 version 17.5 and later validates the debugging libraries shipped with the .NET Runtime are signed before loading them. If they are unsigned, Visual Studio shows an error similar to the following:
```

File: docs/workflow/debugging/coreclr/debugging-runtime.md [hunk 2]
```diff
@@ -223,15 +223,19 @@ Native C++ code is not everything in our runtime. Nowadays, there are lots of st
 
 #### Resolving Signature Validation Errors in Visual Studio
 
-Starting with Visual Studio 2022 version 17.5, Visual Studio will validate that the debugging libraries that shipped with the .NET Runtime are correctly signed before loading them. If they are unsigned, Visual Studio will show an error like:
+Visual Studio 2022 version 17.5 and later validates the debugging libraries shipped with the .NET Runtime are signed before loading them. If they are unsigned, Visual Studio shows an error similar to the following:
```
Comment (Rick-Anderson) [inline docs/workflow/debugging/coreclr/debugging-runtime.md:226]:
```suggestion
Visual Studio 2022 version 17.5 and later validates that the debugging libraries shipped with the .NET Runtime are signed before loading them. If they are unsigned, Visual Studio shows an error similar to the following:
```

File: docs/workflow/debugging/coreclr/debugging-runtime.md [hunk 3]
```diff
@@ -223,15 +223,19 @@ Native C++ code is not everything in our runtime. Nowadays, there are lots of st
...
+  * This is the easiest and recommended approach to temporarily disable signature validation.
+  * At the command line, run `set VSDebugger_ValidateDotnetDebugLibSignatures=0` and then start Visual Studio (`devenv.exe`) from the same command prompt.
+  * This setting is only valid for the Visual Studio instance that is started from the command prompt where the environment variable is set.
+1. The [`DOTNET_ROOT` environment variable](https://learn.microsoft.com/dotnet/core/tools/dotnet-environment-variables#dotnet_root-dotnet_rootx86): if Visual Studio is started from a command prompt where `DOTNET_ROOT` is set, it ignores unsigned .NET runtime debugger libraries which are under the `DOTNET_ROOT` directory.
+1. ***NOT RECOMMENDED** Set the `ValidateDotnetDebugLibSignatures` registry key: To disable signature validation on a more permanent basis, set the `Common7\IDE\VsRegEdit.exe set local HKCU Debugger\EngineSwitches ValidateDotnetDebugLibSignatures dword 0` VS registry key. For example, open a Developer Command Prompt and run `Common7\IDE\VsRegEdit.exe set local HKCU Debugger\EngineSwitches ValidateDotnetDebugLibSignatures dword 0`
```
Comment (Rick-Anderson) [inline docs/workflow/debugging/coreclr/debugging-runtime.md:241]:
```suggestion
1. ***NOT RECOMMENDED*** Set the `ValidateDotnetDebugLibSignatures` registry key: To disable signature validation on a more permanent basis, set the `Common7\IDE\VsRegEdit.exe set local HKCU Debugger\EngineSwitches ValidateDotnetDebugLibSignatures dword 0` VS registry key. For example, open a Developer Command Prompt and run `Common7\IDE\VsRegEdit.exe set local HKCU Debugger\EngineSwitches ValidateDotnetDebugLibSignatures dword 0`
```

**Review by gregg-miskelly — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@gregg-miskelly @steveharter can you recommend an approved reviewer?

**Comment by gregg-miskelly** [comment]
@tommcdon Can you review this documentation change for @Rick-Anderson?

**Review by tommcdon — APPROVED** [review]
LGTM, thanks!

**Pull request merged**